manifests/v/ViviCorporation/Vivi/3.6.0/ViviCorporation.Vivi.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ViviCorporation.Vivi
+PackageVersion: 3.6.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+ProductCode: '{E714C6A8-8753-4FA3-ABC9-B3F71BC22842}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.vivi.io/app/ViviSetup64-3.6.0.msi
+  InstallerSha256: B3DADB4E8665FE18D518289C52CD6DE79792DF40BBE6DBA9B1F42FCB383C08DD
+- Architecture: neutral

```

```suggestion
- Architecture: x86
```
This msi probably won't work on ARM64 builds